test/install/2_download_asset_test.sh

```diff
@@ -0,0 +1,43 @@
+. test_harness.sh
```

Comment:
Is this the name we want for this file? Should it be `download_asset_release_test`?